objgraph.py

```diff
@@ -675,6 +676,9 @@ def show_backrefs(objs, max_depth=3, extra_ignore=(), filter=None, too_many=10,
     Use ``extra_info`` (a function taking one argument and returning a
     string) to report extra information for objects.
 
+    Use ``extra_node_attrs`` (a function returning a dict of strings) to add
+    extra attributes to the nodes.
```

Comment:
What arguments does the function get, if any?

Also it might be nice to link to the Graphviz documentation that describes what node attributes can be specified.

Reply:
OK, will do.